src/software/simulation/physics/physics_world_test.cpp

```diff
@@ -3,6 +3,8 @@
 #include <gmock/gmock.h>
 #include <gtest/gtest.h>
 
+#include "shared/2015_robot_constants.h"
+#include "shared/2015_wheel_constants.h"
```

Comment:
nit; double include (I'd do a pass over the whole PR, there are few other places we do this).

Reply:
robot_constants vs wheel_constants